ponyo/similarity_metric_parallel.py

```diff
@@ -0,0 +1,270 @@
+'''
+Author: Alexandra Lee
+Date Created: 30 August 2019
+
+Scripts to compare simulated compendium with simulated compendia with noise added
+'''
+
+import os
+import sys
+import ast
+import pandas as pd
+import numpy as np
+import random
+import glob
+import warnings
+warnings.filterwarnings(action='ignore')
+
+sys.path.append("../")
```

Comment:
You shouldn't need to mess with the pythonpath; cca_core is in the same directory as similarity_metric_paralell